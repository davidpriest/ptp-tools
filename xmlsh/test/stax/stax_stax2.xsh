import commands stax=stax


r=stax:newEventReader(../../samples/data/books.xml)

while [ stax:hasNext($r START_ELEMENT) ] 
do
	
	e=stax:nextEvent($r)
	type=stax:getEventType($e)
	echo event type $type
	echo stax:getName($e)

	
done


stax:closeReader $r 



