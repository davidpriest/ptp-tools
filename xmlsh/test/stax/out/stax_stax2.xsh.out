event type START_ELEMENT
BOOKLIST
event type START_ELEMENT
BOOKS
event type START_ELEMENT
ITEM
event type START_ELEMENT
TITLE
event type START_ELEMENT
AUTHOR
event type START_ELEMENT
PUBLISHER
event type START_ELEMENT
PUB-DATE
event type START_ELEMENT
LANGUAGE
event type START_ELEMENT
PRICE
event type START_ELEMENT
QUANTITY
event type START_ELEMENT
ISBN
event type START_ELEMENT
PAGES
event type START_ELEMENT
DIMENSIONS
event type START_ELEMENT
WEIGHT
event type START_ELEMENT
ITEM
event type START_ELEMENT
TITLE
event type START_ELEMENT
AUTHOR
event type START_ELEMENT
PUBLISHER
event type START_ELEMENT
PUB-DATE
event type START_ELEMENT
LANGUAGE
event type START_ELEMENT
PRICE
event type START_ELEMENT
QUANTITY
event type START_ELEMENT
ISBN
event type START_ELEMENT
PAGES
event type START_ELEMENT
DIMENSIONS
event type START_ELEMENT
WEIGHT
event type START_ELEMENT
ITEM
event type START_ELEMENT
TITLE
event type START_ELEMENT
AUTHOR
event type START_ELEMENT
PUBLISHER
event type START_ELEMENT
PUB-DATE
event type START_ELEMENT
LANGUAGE
event type START_ELEMENT
PRICE
event type START_ELEMENT
QUANTITY
event type START_ELEMENT
ISBN
event type START_ELEMENT
PAGES
event type START_ELEMENT
DIMENSIONS
event type START_ELEMENT
WEIGHT
event type START_ELEMENT
ITEM
event type START_ELEMENT
TITLE
event type START_ELEMENT
AUTHOR
event type START_ELEMENT
PUBLISHER
event type START_ELEMENT
PUB-DATE
event type START_ELEMENT
LANGUAGE
event type START_ELEMENT
PRICE
event type START_ELEMENT
QUANTITY
event type START_ELEMENT
ISBN
event type START_ELEMENT
PAGES
event type START_ELEMENT
DIMENSIONS
event type START_ELEMENT
WEIGHT
event type START_ELEMENT
ITEM
event type START_ELEMENT
TITLE
event type START_ELEMENT
AUTHOR
event type START_ELEMENT
PUBLISHER
event type START_ELEMENT
PUB-DATE
event type START_ELEMENT
LANGUAGE
event type START_ELEMENT
PRICE
event type START_ELEMENT
QUANTITY
event type START_ELEMENT
ISBN
event type START_ELEMENT
PAGES
event type START_ELEMENT
DIMENSIONS
event type START_ELEMENT
WEIGHT
event type START_ELEMENT
ITEM
event type START_ELEMENT
TITLE
event type START_ELEMENT
AUTHOR
event type START_ELEMENT
PUBLISHER
event type START_ELEMENT
PUB-DATE
event type START_ELEMENT
LANGUAGE
event type START_ELEMENT
PRICE
event type START_ELEMENT
QUANTITY
event type START_ELEMENT
ISBN
event type START_ELEMENT
PAGES
event type START_ELEMENT
DIMENSIONS
event type START_ELEMENT
WEIGHT
event type START_ELEMENT
CATEGORIES
event type START_ELEMENT
CATEGORY
event type START_ELEMENT
CATEGORY
event type START_ELEMENT
CATEGORY
